docs/en-US/component/scrollbar.md

```diff
@@ -60,22 +60,21 @@ scrollbar/manual-scroll
 ### Events
 
 | Name   | Description                                           | Type                                                             |
-| ------ | ----------------------------------------------------- | ---------------------------------------------------------------- |
+|--------|-------------------------------------------------------|------------------------------------------------------------------|
 | scroll | triggers when scrolling, return distance of scrolling | ^[Function]`({ scrollLeft: number, scrollTop: number }) => void` |
 
 ### Slots
 
 | Name    | Description               |
-| ------- | ------------------------- |
+|---------|---------------------------|
 | default | customize default content |
 
 ### Exposes
 
 | Method        | Description                                | Type                                                                       |
-| ------------- | ------------------------------------------ | -------------------------------------------------------------------------- |
+|---------------|--------------------------------------------|----------------------------------------------------------------------------|
 | handleScroll  | handle scroll event                        | ^[Function]`() => void`                                                    |
 | scrollTo      | scrolls to a particular set of coordinates | ^[Function]`(options: ScrollToOptions \| number, yCoord?: number) => void` |
 | setScrollTop  | Set distance to scroll top                 | ^[Function]`(scrollTop: number) => void`                                   |
 | setScrollLeft | Set distance to scroll left                | ^[Function]`(scrollLeft: number) => void`                                  |
-| update        | update scrollbar state manually            | ^[Function]`() => void`                                                    |
-| wrapRef       | scrollbar wrap ref                         | ^[Object]`Ref<HTMLDivElement>`                                             |
+| update        | update scrollbar state manually            | ^[Function]`() => void`                                                    |
```

Comment:
## Question

Why is it exposed in code but deleted in document?

```vue
// scrollbar.vue
defineExpose({
  /** @description scrollbar wrap ref */
  wrapRef, // here
  /** @description update scrollbar state manually */
  update,
  /** @description scrolls to a particular set of coordinates */
  scrollTo,
  /** @description set distance to scroll top */
  setScrollTop,
  /** @description set distance to scroll left */
  setScrollLeft,
  /** @description handle scroll event */
  handleScroll,
})
```

Reply:
@ryuhangyeong, the code has not been published yet, so we think it is better that we delete it in the docs first, another reason is that we need to fix some doc issues and do a doc release, and we don't really want to misguide our users.

Reply:
@jw-foss Thanks for the reply~